### ISSUE TITLE
Volumetric::cone might be wrong in 3D?

### ISSUE THREAD

**Issue by quadrupleslap** [sebcrozet/nphysics#120]
The angular inertia formula that you're using in volumetric/volumetric_cone.rs looks like the one on Wikipedia, which is used about the apex. But if I understand it correctly, the angular inertia that we generally want is the one about the center of mass.

(I don't know much physics though, so there's a good chance that I'm wrong.)

**Comment by sebcrozet** [comment]
That's correct, there is an error here. Though cones (and cylinders) are no longer supported by nphysics 0.8 (for accuracy reason. Convex polyhedral approximations are to be preferred), and the `volumetric/volumetric_cone.rs` file is not even compiled so it should just be removed.
